### DISCUSSION TITLE
【924970】喋血复仇 （本体）

### DISCUSSION THREAD

**Discussion by weilovelili** [pjy612/SteamManifestCache#333] [Share]
appid：924970
key：
```
924971;783767a483d935235bc2a46262a58b4acc3524bdf6d52116f090fcccc0a89024
```
code：2E2C741D64BBE1C5
email：dyanr@outlook.com

**Comment by ZEERDEER** [comment]
请按格式分享 #26 

DLC Depot ID 查阅以下链接

https://steamdb.info/app/924970/depots/?branch=public

**Reply by weilovelili** [reply]
我看steamdb上是924970 我本地是924971？会不会已经包含了
![image](https://github.com/pjy612/SteamManifestCache/assets/156031610/fdcb7532-2d40-4080-9db9-66f1f06df0d6)
是这种情况

**Reply by weilovelili** [reply]
你看怎样

**Comment by pjy612** [comment]
请修正密钥格式

**Reply by weilovelili** [reply]
这样吗

**Reply by weilovelili** [reply]
这会对了吧
